src/rtextures.c

```diff
@@ -1630,6 +1630,241 @@ Image ImageTextEx(Font font, const char *text, float fontSize, float spacing, Co
     return imText;
 }
 
+Image ImageFromChannel(Image image, int selected_channel, float threshold)
```

Comment:
I think the `threshold` parameter could be un-intuitive and managed in user-side when the channel is extracted, please, could you remove it?